monitoring/using-rfhe.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+[id="using-rfhe"]
+= Monitoring Redfish hardware events
+include::modules/common-attributes.adoc[]
+:context: using-rfhe
+
+toc::[]
```

Comment:
remove this from here.

Reply:
Yes, monitoring I think is a much better home for this section